project-koku/koku
# [COST-3535] Skip markup calculation when no markup

## Jira Ticket

[COST-3535](https://issues.redhat.com/browse/COST-3535)

## Description

This change will not run OpenShift markup calculations when no markup is in the cost model. 

## Testing

1. make create-test-customer and load test customer data
2. Run http://127.0.0.1:5042/api/cost-management/v1/update_cost_model_costs/?provider_uuid=6764b510-1807-470f-a833-4058b4fa50d9&schema=org1234567&start_date=2023-02-01&end_date=2023-02-22 replacing provider uuid with your OCP on prem provider UUID
3. Watch worker logs and see `No markup to calculate for <provider_uuid>."` logged

## Notes

...
